generators/generator-eclipseditto/src/main/java/org/eclipse/vorto/codegen/ditto/schema/tasks/template/DittoStructureTemplate.xtend

```diff
@@ -30,13 +30,13 @@ class DittoStructureTemplate implements IFileTemplate<InformationModel> {
 	override getPath(InformationModel context) {
 		return null;
 	}
-	
+
 	override getContent(InformationModel model, InvocationContext context) {
 		'''
 {
+  "definition": ["«model.namespace»:«model.name»:«model.version»"],
   "attributes": {
- 	"thingName": "«model.displayname»",
- 	"definition": "«model.namespace»:«model.name»:«model.version»"
+ 	"thingName": "«model.displayname»"
```

Comment:
"thingName" could be confusing for the user as Ditto names the part of the thingId after the `:` also "thingName", at least in the model classes.
Maybe better use "displayName" or even "modelDisplayName".

Reply:
modelDisplayName does in fact describe this field more accurately, especially, since the value of this field will only be stored as additional attribute in Ditto. 
https://github.com/eclipse/vorto/issues/2215